cmd/dfget/app/root.go

```diff
@@ -120,6 +124,14 @@ func checkParameters() error {
 	return nil
 }
 
+// getDefaultConfig returns the default configuration of dfget
+func getDefaultConfig() (interface{}, error) {
+	if _, err := initProperties(); err != nil {
+		return nil, err
+	}
+	return cfg, nil
```

Comment:
why return `cfg` here? And I think it's not ready for dfget to get the default config easily without `viper`. I think maybe you can using viper to manage the configurations for dfget firstly. Ref #959 .

Reply:
I will do it ASAP.